docs/Configuration.md

```diff
@@ -70,8 +70,16 @@ countsSettings {
   password: "admin"
 }
 
-```
-In the next sections we'll look at 
+```             
+###Default configurations
+Default configuration for UI is in [defaultConfig.conf](https://github.com/TouK/nussknacker/blob/staging/ui/server/src/main/resources/defaultConfig.conf).
+We don't use ```reference.conf``` at the moment, as classloaders of model and ui are not separated, and we don't want UI config to be passed to model. 
+
+Default configuration of models is taken from ```model.conf``` files in model jar (see e.g. [model.conf](https://github.com/TouK/nussknacker/blob/staging/engine/flink/generic/src/main/resources/model.conf)).
```

Comment:
I don't exactly get it. How someone can override this default config? If it is really default config that can be overriden maybe better will be call it `defaultModelConfig.conf`?

Reply:
I added more explanation + example

Reply:
I renamed as you suggested